content/02.main-text.md

```diff
@@ -257,3 +263,25 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Our process represents an early step toward open massively collaborative reviews, and there are certainly aspects that can be improved.
 We invite the scientific community to adapt and build upon our experience and open software.
+
+### The open science movement
+
+Science is undergoing a transition towards openness.
+The internet provides a global information commons, where scholarship can be publicly shared at a minimal cost.
+For example, open access publishing provides an economic model that encourages maximal dissemination and reuse of scholarly articles [@doi:10.12688/f1000research.8460.3].
+Pressure from funders, libraries, self-archiving, and piracy are set to cement open access as the primary model for publishing the scientific record [@doi:10.7717/peerj.4375; @doi:10.7554/eLife.16800; @doi:10.7554/eLife.32822; @doi:10.1038/d41586-018-05754-1].
+However, open science extends beyond just open access to journal articles.
+Open licensing solves legal barriers to content reuse, enabling any type of scholarly output to become part of the commons [@doi:10.1109/MCSE.2009.19; @doi:10.1038/536016a].
+
+The push for increased reproducibility and the opportunity to reuse data and code for new investigations has begot a movement to make all research outputs public, unless there are bonafide privacy or security concerns [@doi:10.1126/science.aah6168; @doi:10.1038/nature10836; @doi:10.1371/journal.pbio.1001195].
```

Comment:
This paragraph is the most relevant in my opinion.  With the one exception noted below, I think we should keep all of it.